### ISSUE TITLE
"500 Internal Server Error" when adding a new issue

### ISSUE THREAD

**Issue by topkool** [luisfontes19/redmine_stats#14]
When I add a new issue, I have to proceed in 2 steps:

1. I create the issue, and I get a "500 internal server error" message.

Log:
~~~
Started POST "/projects/terminal/issues" for <masked> at 2015-04-20 13:16:53 +0200
Processing by IssuesController#create as HTML
  Parameters: {"utf8"=>"?", "authenticity_token"=>"<masked>", "issue"=>{"is_private"=>"0", "tracker_id"=>"2", "subject"=>"Empêcher et gérer...", "description"=>"L'utilisation d'une ...", "status_id"=>"1", "priority_id"=>"2", "assigned_to_id"=>"9", "category_id"=>"28", "fixed_version_id"=>"5", "parent_issue_id"=>"", "start_date"=>"2015-04-20", "due_date"=>"", "custom_field_values"=>{"10"=>"I5"}, "project_ids"=>["", "2"], "answers_on_secondary_projects"=>"1"}, "issue_template"=>"", "was_default_status"=>"1", "commit"=>"Create", "project_id"=>"<masked>"}
  Current user: <masked> (id=8)
  Rendered mailer/issue_add.text.erb within layouts/mailer (33.5ms)
Completed 500 Internal Server Error in 272ms

ActionView::Template::Error (can not load translations from /usr/share/redmine-3.0.1/plugins/redmine_stats/config/locales/fr.yml: #<Psych::SyntaxError: (/usr/share/redmine-3.0.1/plugins/redmine_stats/config/locales/fr.yml): found character that cannot start any token while scanning for the next token at line 3 column 1>):
    1: <%= l(:text_issue_added, :id => "##{@issue.id}", :author => @issue.author) %>
    2:
    3: ----------------------------------------
    4: <%= render :partial => 'issue', :formats => [:text], :locals => { :issue => @issue, :users => @users, :issue_url => @issue_url } %>
  lib/redmine/i18n.rb:159:in `init_translations'
  lib/redmine/i18n.rb:173:in `lookup'
  lib/redmine/i18n.rb:30:in `l'
  app/views/mailer/issue_add.text.erb:1:in `_5c9ab911d6567e8f18a3673d36595c37'
  app/models/mailer.rb:435:in `block in mail'
  app/models/mailer.rb:434:in `mail'
  app/models/mailer.rb:51:in `issue_add'
  app/models/mailer.rb:447:in `initialize'
  config/initializers/10-patches.rb:154:in `deliver'
  app/models/mailer.rb:61:in `block in deliver_issue_add'
  app/models/issue.rb:872:in `each_notification'
  app/models/mailer.rb:60:in `deliver_issue_add'
  app/models/issue.rb:1553:in `send_notification'
  app/models/issue.rb:174:in `create_or_update'
  app/controllers/issues_controller.rb:141:in `create'
~~~

2. I refresh the page, and now, the issue is created without issue

**Comment by nothing-fr** [comment]
try my fork, i have corrected the french translation.

https://github.com/nothing-fr/redmine_stats

**Comment by luisfontes19** [comment]
Solved in 0088e2d0c62c76c14aea43aad2efdb65aae034e6
Thanks nothing-fr
